### PR TITLE
JW-1290 Enable cloning Juniper/contrail-windows-docker-driver

### PR DESCRIPTION
**Pull request by krdln**
This time, actually put it in *src/github.com/Juniper/contrail-windows-docker-driver* instead of *src/github.com/codilime/contrail-windows-docker*.

Unfortunately, both our tests and the Wix config assumes the name of binary is just *contrail-windows-docker*, which is not the case with new repo. Temporarily, I've added a `-o contrail-windows-docker.exe` to work around this issue, but we'll need to make a decision what to do with this renaming.

### CODE REVIEW

**Comment by krdln** [comment]
I've tested on a job with the same length as `ci-contrail-win-server2016-production` (37 chars) and it failed due to longer paths.

**Comment by krdln** [comment]
New jobs have sufficiently small length, so it should work.

**Comment by krdln** [comment]
cc @m-kostrzewa wrt. `GOPATH`
Rebased.
[Test log](http://10.84.12.26:8080/job/WinContrail/job/winci-server2016-mkrmkr/51/console)

File: CIScripts/Build/Zuul.ps1 [hunk 1]
```diff
@@ -35,10 +36,11 @@ function Clone-ZuulRepos {
             zuul-cloner.exe @ZuulClonerOptions @ProjectList
         }
         
-        # TODO: Use Juniper repo: git clone contrail-windows-docker-driver.github.com:Juniper/contrail-windows-docker-driver.git
+        $DriverClonePath = $DriverSrcPath -replace "github.com/", "github.com:"
```
Comment (kaweue) [inline CIScripts/Build/Zuul.ps1]: This feature seems to be unrelated to cloning from zuul/gerrit. To avoid reader's confusion it should be done directly in Build.ps1. Clone-Repos function might be a useful tool.

File: CIScripts/Build/ProdCI.ps1 [hunk 2]
```diff
@@ -24,8 +24,8 @@ function Get-ProductionRepos {
 
     $Repos = @{
         # Windows docker driver is not on gerrit (yet)
-        "contrail-windows-docker" = [Repo]::new("https://github.com/Juniper/contrail-windows-docker-driver/",
-                                                "master", "master", "src/github.com/codilime/contrail-windows-docker");
+        "contrail-windows-docker" = [Repo]::new("contrail-windows-docker-driver.github.com:Juniper/contrail-windows-docker-driver.git",
```
Comment (kaweue) [inline CIScripts/Build/ProdCI.ps1]: We think that function Get-ProductionRepos can be removed.

**Review by kaweue — CHANGES_REQUESTED** [review]
PM, JI

**Review by jacek-izykowski — CHANGES_REQUESTED** [review]
PM, JI

**Review by jacek-izykowski — APPROVED** [review]

**Review by kaweue — APPROVED** [review]
